hazelcast/src/test/java/com/hazelcast/internal/serialization/impl/compact/integration/ClientCompactSplitBrainTest.java

```diff
@@ -102,4 +116,100 @@ public void testLocalSchemasAreSent_whenClientReconnectsToOtherHalf() {
             CompactTestUtil.assertSchemasAvailable(Collections.singletonList(reconnectedMember), EmployeeDTO.class);
         });
     }
+
+    @Test
+    public void testSchemaReplicationRetried_whenClientIsConnectedToBothHalvesOfTheSplit() {
```

Comment:
This test takes 17 seconds on my local. Is this class a `QuickTest` after this test is added?

Reply:
I made the test faster, and increased the max retry count as well, just to be more safe